src/alasqlparser.jison

```diff
@@ -1460,9 +1460,43 @@ LogicValue
 
 StringValue
 	: STRING
-		{ $$ = new yy.StringValue({value: $1.substr(1,$1.length-2).replace(/(\\\')/g,"'").replace(/(\'\')/g,"'")}); }
+		{ 
+			var str = $1.substr(1,$1.length-2).replace(/(\'\')/g,"'");
+			str = str.replace(/\\(.)/g, function(match, char) {
+				switch(char) {
+					case 'n': return '\n';
+					case 't': return '\t';
+					case 'r': return '\r';
+					case 'b': return '\b';
+					case 'f': return '\f';
+					case 'v': return '\v';
+					case '\\': return '\\';
```

Comment:
Please check if this is being solved via https://github.com/AlaSQL/alasql/pull/2335